### PR TITLE
[Automation][NFS-Ganesha][HA]Add testcase

### PR DESCRIPTION
**Pull request by MohitBis**
[Automation][NFS-Ganesha][HA]Add testcase
Scenario:
1. Deploy nfs-ganesha with ha using spec file.
2. Rm's from client C1 and small files and large files creation from another client C2, and lookups from client C3 + Node reboot
3. HA Run linux untars, mkdirs and touch's and perform find's,ls -laRt and du -sh from another 3 clients in loop

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-S2IHNE/

**Review by pranavprakash20 — APPROVED** [review]

File: tests/nfs/nfs_ha_readdir_operation.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        for Thread_operation in Thread_operations:
+            Thread_operation.start()
+            sleep(3)
+
+        # Wait to complete operations
+        for Thread_operation in Thread_operations:
+            Thread_operation.join()
+
+        sleep(30)
+        # Reboot all clients
```
Comment (msainiRedhat) [inline tests/nfs/nfs_ha_readdir_operation.py:137]: Need to reboot server node having VIP assigned

**Review by msainiRedhat — CHANGES_REQUESTED** [review]

**Comment by MohitBis** [comment]
Log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-0AFWSI/nfs-ganesha-ha_readdir_operations_0.log

**Review by pranavprakash20 — APPROVED** [review]
LGTM

File: tests/nfs/nfs_ha_readdir_operation.py (hunk 1 above)
Comment (MohitBis) [reply tests/nfs/nfs_ha_readdir_operation.py]: Updated

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Pull request merged**